pkg/arvo/sys/lull.hoon

```diff
@@ -1813,17 +1814,130 @@
   +$  verb  ?(%odd)
   ::
   ::  +agent: app core
+  +$  agent-wefts  ?([%zuse %416])
   ::
-  ++  agent
+  ++  agent-any  $%  [[%zuse %416] agent-416]
+                 ==
+  ++  agent  agent-416
+  ::
+  ++  agent-416
     =<  form
     |%
     +$  step  (quip card form)
     +$  card  (wind note gift)
     +$  note
       $%  [%agent [=ship name=term] =task]
-          [%arvo note-arvo]
+          [%arvo userspace-note]
           [%pyre =tang]
       ==
+    +$  userspace-gift
+      $%  $>(%wake gift:behn)
+          $>(%heck gift:behn)
+          ::
+          $>(%wris gift:clay)
+          $>(%writ gift:clay)
+          $>(%tire gift:clay)
+          $>(%hill gift:clay)
+          $>(%mere gift:clay) :: kiln
+          $>(%cruz gift:clay)
+          $>(%croz gift:clay)
+          ::
+          $>(%blit gift:dill)
+          ::
+          $>(%bound gift:eyre)
+          ::
+          $>(%unto gift:gall)
+          ::
+          $>(%http-response gift:iris)
+          ::
+          $>(%private-keys gift:jael)
+          $>(%public-keys gift:jael)
+          $>(%turf gift:jael)
+          ::
+          $>(%arow gift:khan)
+      ==
+    +$  userspace-note
+      $%  $:  %a
+        $%  $>(%sift task:ames)
+            $>(%spew task:ames) :: set verbosity (we want this?)
+            $>(%cork task:ames) :: gall does this
```

Comment:
I'm not sure whether userspace should ever be allowed to %cork.  I can see the use case for recovery commands, although I'd be inclined to expose those commands as ames tasks rather than letting userspace cork willy-nilly.